templates/web/base/waste/bulky/choose_date.html

```diff
@@ -0,0 +1,15 @@
+[% SET title = form.title ~%]
```

Comment:
This template is identical to waste/index.html - can we not use the default?

Reply:
The identical template was deliberate to make it easier for someone else to pick up the implementation later on. Perhaps just a one-liner like `[% INCLUDE 'waste/index.html' %]` would be OK?

Reply:
I guess, but I'd hope that for normal form pages, there wouldn't be a need for a custom HTML page at all, it doesn't look like there generally is.

Reply:
Sorry if this wasn't clear, reading it back - I meant that the default template is picked up as a fallback automatically, so I'm not sure specifying templates here in advance makes it easier to pick up later on, because there shouldn't (hopefully/ideally) be anything that needs doing except for very custom pages (like eg the summary page).

Reply:
Gotcha. Will update accordingly.